waterlog/src/Components/FaultySensor.js

```diff
@@ -43,13 +44,15 @@ class FaultySensor extends Component {
 		});
 	}
 
+	segmentMap = <div className="segment-map"><Map height="400px"/><hr /></div>;
```

Comment:
If you declare another style set like below: then you should not need to specify the map's height 
```
div.segment-map Map {
  height: 400px;
}
```

Reply:
just test it first obviously

Reply:
Note this would apply to all of those cases. So I see that later on it is specified as `<Map height="300px"/>` if that is intentional then dont use the above method. 
And you can then just resolve this comment

Reply:
I need the map with different sized on different pages. So yes it is intentional